components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -106,6 +109,9 @@ public String getClassName() {
         return "APIProvider";
     }
 
+    // API definitions from swagger v2.0
+    static APIDefinitionFromSwagger20 definitionFromSwagger20 = new APIDefinitionFromSwagger20();
```

Comment:
Is it intentional that this variable was assigned its access modifier to be default? If not, please fix.